### ISSUE TITLE
Enhancement: Add capability for a string to be included in the file name e.g. 2021-07-24-weekly

### ISSUE THREAD

**Issue by hiveai** [liamcain/obsidian-periodic-notes#59]
It would be great if the file naming allowed arbitrary text to be included along with the date info e.g. YYYY-MM-DD-"MyText"

**Comment by svenluijten** [comment]
That's already possible. Surround the (arbitrary) text you want to be included in brackets (so your example would become `YYYY-MM-DD-[MyText]`), or escape _all_ characters: `YYYY-MM-DD-\M\y\T\e\x\t`.

**Comment by hiveai** [comment]
That's awesome.  Thanks!

What do you think about adding a row in the format table here: https://momentjs.com/docs/#/displaying/format/

It could be something like arbitrary text => Surround arbitrary text with square brackets.  
If something like that had been in the formatting table I would have seen it and just used it.

Cheers!

**Comment by svenluijten** [comment]
@hiveai MomentJS is the library that most Obsidian plugins use for formatting dates; it's not within the scope of this project (or Obsidian) to change the MomentJS documentation unfortunately 🙂

**Comment by hiveai** [comment]
right...I was looking at this again, and noticed that the help just calls out to the Moment.js docs. I'll file an enhancement request to the docs over there.

**Comment by liamcain** [comment]
Thanks for the assist @svenluijten.

The periodic notes docs could definitely use a bit of a revamp. Doesn't hurt to mention how to escape characters in the plugin readme too.
